rest/handler.go

```diff
@@ -440,11 +442,7 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []Permission, r
 					// _default doesn't exist for a non-default scope - so make it a required element if it's ambiguous
 					return base.HTTPErrorf(http.StatusBadRequest, "Ambiguous keyspace: %s.%s", base.MD(keyspaceDb), base.MD(base.StringDefault(keyspaceScope, "")))
 				}
-				// one collection only - use that one
-				for collectionName := range scope.Collections {
-					keyspaceCollection = &collectionName
-					break
-				}
+				keyspaceScope = base.StringPtr(base.DefaultCollection)
```

Comment:
I think this was supposed to be collection?

```suggestion
				keyspaceCollection = base.StringPtr(base.DefaultCollection)
```